docs/cookbook/entities/custom-translatable-model.rst

```diff
@@ -486,14 +501,14 @@ should be also included in the ``app/config/routing.yml``.
         resource: 'routing/admin.yml'
         prefix: /admin
 
-9. Add entity administration to the admin menu
+12. Add entity administration to the admin menu
 ----------------------------------------------
```

Comment:
missing some `---`, here and in 13. :) the underline has to be same length as the text.